Modules/gcmodule.c

```diff
@@ -299,7 +299,7 @@ gc_list_size(PyGC_Head *list)
 }
 
 /* Append objects in a GC list to a Python list.
- * Return 0 if all OK, < 0 if error (out of memory for list).
+ * Return 0 if all OK, < 0 if error (out of memory for list)deduce_unreachable.
```

Comment:
`deduce_unreachable` looks like a mistake here (has nothing to do with the comment).

Reply:
Sorry about this, I was coding in a different editor and I clearly don't manage it perfectly :(